monoloco/activity.py

```diff
@@ -115,7 +115,7 @@ def is_raising_hand(kp):
     if is_right_risen:
         return 'right'
 
```

Comment:
I mean you just remove the statement return None (completely, remove also 'return'), and it's the same thing :)